### DISCUSSION TITLE
Reolink Lumus IPC_325C7.705_21052800.IMX307.2MP.WIFI7601.REOLINK firmware

### DISCUSSION THREAD

**Discussion by zd3sf** [AT0myks/reolink-fw-archive#6] [Firmware sharing]
[IPC_325C7.705_21052800.IMX307.2MP.WIFI7601.REOLINK.zip](https://github.com/AT0myks/reolink-fw-archive/files/11149143/IPC_325C7.705_21052800.IMX307.2MP.WIFI7601.REOLINK.zip)

**Comment by AT0myks** [comment]
Added, thanks.
